pytools/tag.py

```diff
@@ -1,5 +1,8 @@
 from dataclasses import dataclass
 from typing import Tuple, Any, FrozenSet, Union, Iterable, TypeVar
+import importlib
+
+from lark import Lark, Transformer
```

Comment:
Could you make it so that `pytools.tag` is still importable without `lark`?

Reply:
Yep, see ff625d904bf91752d6c0e5dc1b4679e652c150a4.